roles/deploy_olm_operator/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 oc_bin_path: oc
 openshift_namespace: test-operator
 current_channel: alpha
-operator_work_dir: /home/jenkins/agent/test-operator
```

Comment:
I would remove this change from the patch.
I asked Scott to unify names in initial pull request. because on some places we use `/home/jenkins/agent/test-operator` and on other `home/jenkins/test-operator`. It should not case any isseu in downstream.